lib/node/process.dart

```diff
@@ -17,6 +17,7 @@ extension type Process(JSObject obj) {
   external JSObject get _env;
 
   /// Read the environment variable [variable].
+  // TODO(devoncarew): move away from using dart:js_util `getProperty`.
```

Comment:
@srujzs - quick interop question. If I have a JS object, how to I access named fields (like `'PWD'` or `'PATH'`).

Context: https://nodejs.org/api/process.html#processenv

Reply:
ah, `JSObjectUnsafeUtilExtension.getProperty` :)

Reply:
Should the docs for https://api.dart.dev/stable/latest/dart-js_interop_unsafe/JSObjectUnsafeUtilExtension.html be updated? It says `but takes and returns a Dart value` but the sig indicates it takes a dart value and returns a JSAny.

Reply:
Yup, or just `[]`. You could also add an extension type for `_env` and add `external` fields with the expected names.

Thanks for the catch! https://dart-review.googlesource.com/c/sdk/+/409480